.cspell.json

```diff
@@ -78,6 +78,7 @@
         "keyserver",
         "levelno",
         "linuxize",
+        "manulipulations",
```

Comment:
heh. seems like we should file an issue to get this fixed in the edk2 tool-lib. Not a blocker for this PR, given that it's caused by upstream naming.

Reply:
Yeah, it was disappointing to see that. Tracking issue filed here - https://github.com/tianocore/edk2-pytool-library/issues/724. Resolving this thread.